normandy/recipes/admin.py

```diff
@@ -45,11 +43,30 @@ class RecipeAdmin(NonSortableParentAdmin):
 
 @admin.register(models.Action)
 class ActionAdmin(admin.ModelAdmin):
-    list_display = ['name', 'implementation_hash']
+    form = ActionAdminForm
+    list_display = ['name', 'implementation_hash', 'in_use']
     fieldsets = [
         [None, {
-            'fields': ['name', 'implementation_hash', 'implementation']
+            'fields': [
+                'name',
+                'implementation_hash',
+                'implementation',
+                'arguments_schema_json',
+                'recipe_list',
+            ]
         }],
     ]
 
-    readonly_fields = ['implementation_hash']
+    readonly_fields = ['implementation_hash', 'recipe_list']
+
+    def recipe_list(self, action):
+        """List all recipes that the action is being used by."""
+        return render_to_string('admin/field_recipe_list.html', {
+            'recipes': action.recipes_used_by.order_by('name'),
+        })
+    recipe_list.short_description = 'Used in Recipes'
+
+    def in_use(self, action):
+        """Callable on the ModelAdmin so we can set boolean=True."""
```

Comment:
It means that we can't just use the `in_use` property on the model because it won't show as the nice boolean checkmark or X on the admin with the `boolean` property set on it, and we can't set it on the property itself.

I'll try and be clearer about that.